db/migrate/20150227185631_add_users_state_flag.rb

```diff
@@ -0,0 +1,15 @@
+class AddUsersStateFlag < ActiveRecord::Migration
+  def up
+    add_column :users, :state, :string, default: 'temp'
+    execute "update users set state='temp' where is_temp='t'"
```

Comment:
@Dantemss I am no sql guy - can you make sure to review this line for working in sqlite and postgres?